src/guides/v2.2/frontend-dev-guide/translations/theme_dictionary.md

```diff
@@ -76,11 +76,17 @@ You can translate the same element in different ways for different modules:
 ## Additional information
 
 -  [Translations overview]
+-  [Translate theme strings]
```

Comment:
Don't add links to topics that are in the same section of the left navigation. Remove those links, but keep the one to the translation project.